tests/validators/tests.py

```diff
@@ -205,6 +205,11 @@
 
     (URLValidator(EXTENDED_SCHEMES), 'file://localhost/path', None),
     (URLValidator(EXTENDED_SCHEMES), 'git://example.com/', None),
+    (URLValidator(EXTENDED_SCHEMES), 'ssh://example.com/', None),
```

Comment:
What's the relevance of this test for ssh://? It seems to already pass before this commit.